wd_helper.au3

```diff
@@ -751,7 +751,7 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 		$iErr = @error
 	WEnd
 
-	Local $iReturn = (($iErr) ? (0) : (1))
+	Local $iReturn = ($iErr) ? (0) : (1)
```

Comment:
Just curious... Do you realize that the remaining parentheses aren't needed?

Reply:
Yes.
But my experience says that in such case:
```autoit
Local $sMessage = '$sCommand = ' & $sCommand & ' : Result = ' & ((IsArray($vResult)) ? ("(array)") : ($vResult))
```

it is recomended, 

especailly in such scenario
```autoit
Local $sMessage = '$sCommand = ' & $sCommand & ' : Result = ' & ((IsArray($vResult)) ? ("(array)") : ($vResult)) & ' some sufix.......'
```

for this reason in `SciTE4AutoIt3` there is added `ternary` abbreviation which gives such result
```autoit
((@compiled) ? (True) : (False))
```
And I use them all the time.